patches/0012-remove-long-path-support-hack.patch

```diff
@@ -0,0 +1,177 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: qmuntal <quimmuntal@gmail.com>
+Date: Tue, 12 Mar 2024 12:26:53 +0100
+Subject: [PATCH] remove long path support hack
+
+This commit backports https://github.com/microsoft/go/pull/1171 and
```

Comment:
It looks like the #1171 changes aren't here?

Reply:
Wops, adding it now.